src/app/services/login-services/login.service.ts

```diff
@@ -0,0 +1,53 @@
+import { environment } from "src/environments/environment";
+import { loginEnviroment } from "../../../environments/loginSettings";
+
+import { Injectable } from "@angular/core";
+import {
+  HttpClient,
+  HttpHeaders,
+  HttpErrorResponse
+} from "@angular/common/http";
+import { Observable, of } from "rxjs";
+import { map, catchError, tap } from "rxjs/operators";
+
+const httpOptions = {
+  headers: new HttpHeaders({
+    "Content-Type": "application/json"
+  })
+};
+
+@Injectable({
+  providedIn: "root"
+})
+export class LoginService {
+  constructor(private http: HttpClient) {}
+
+  verifyUserCredentials(userDetails): Observable<any> {
+    return this.http
+      .post<any>(
+        loginEnviroment.loginBaseUrl +
```

Comment:
create a URL variable, and use it here instead.

Reply:
Done